### PR TITLE
feat(error): #1422 Error module and 404 page

### LINKED ISSUE #1422: Error page (404)
Create error module and catch all UI routing that is not matched and create a `404` page using the mascot with ears down...

### PR DESCRIPTION
**Pull request by eddiejaoude**
closes #1422 

### Notes

A summary of what was achieved in this PR

- [ ] Error module
- [ ] 404 error page

Screenshot 2019-08-29 08 17 17

### CODE REVIEW

File: web/src/app/error/error.module.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Core components
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+
+// Application component
+import { SharedModule } from '@app/shared/shared.module';
```
Comment (webkhushboo) [inline web/src/app/error/error.module.ts]: Here we can use `@shared` instead of `@app/shared`

Comment (eddiejaoude) [reply web/src/app/error/error.module.ts]: Good spot 👍

Comment (eddiejaoude) [reply web/src/app/error/error.module.ts]: I just copied that part 😂 😜 - I didn't spot it 🤦‍♂ . I will update

**Comment by eddiejaoude** [comment]
Great spot on the scroll. I think I fixed it. Please review again 🤓

File: web/src/app/error/error.module.ts (hunk 1 above)
Comment (webkhushboo) [reply web/src/app/error/error.module.ts]: Yes it was my mistake :( . I noticed now so I commented

**Comment by webkhushboo** [comment]
Looks good now :)

**Pull request merged**